Sources/Orbit/Components/InputField.swift

```diff
@@ -28,40 +28,47 @@ public struct InputField: View {
     let autocapitalization: UITextAutocapitalizationType
     let isAutocompleteEnabled: Bool
     let isSecure: Bool
+    let passwordStrength: PasswordStrength
     let message: MessageType
     let onEditingChanged: (Bool) -> Void
     let onCommit: () -> Void
     let suffixAction: (() -> Void)?
 
     public var body: some View {
         FormFieldWrapper(label, message: message, messageHeight: $messageHeight) {
-            InputContent(
-                prefix: prefix,
-                suffix: suffix,
-                state: state,
-                message: message,
-                isEditing: isEditing,
-                suffixAction: suffixAction
-            ) {
-                HStack(spacing: 0) {
-                    input
-                        .textFieldStyle(TextFieldStyle(leadingPadding: 0))
-                        .autocapitalization(autocapitalization)
-                        .disableAutocorrection(isAutocompleteEnabled == false)
-                        .textContentType(textContent)
-                        .keyboardType(keyboard)
-                        .font(.orbit(size: Text.Size.normal.value, weight: .regular))
-                        .accentColor(.blueNormal)
-                        .background(textFieldPlaceholder, alignment: .leading)
-                        .disabled(state == .disabled)
-                    if isSecure {
-                        securedSuffix
-                    } else {
-                        clearButton
+            VStack(alignment: .leading, spacing: .xxSmall) {
+                InputContent(
+                    prefix: prefix,
+                    suffix: suffix,
+                    state: state,
+                    message: message,
+                    isEditing: isEditing,
+                    suffixAction: suffixAction
+                ) {
+                    HStack(spacing: 0) {
+                        input
+                            .textFieldStyle(TextFieldStyle(leadingPadding: 0))
+                            .autocapitalization(autocapitalization)
+                            .disableAutocorrection(isAutocompleteEnabled == false)
+                            .textContentType(textContent)
+                            .keyboardType(keyboard)
+                            .font(.orbit(size: Text.Size.normal.value, weight: .regular))
+                            .accentColor(.blueNormal)
+                            .background(textFieldPlaceholder, alignment: .leading)
+                            .disabled(state == .disabled)
+                        if isSecure {
+                            securedSuffix
+                        } else {
+                            clearButton
+                        }
                     }
                 }
+
+                PasswordStrengthIndicator(passwordStrength: passwordStrength)
+                    .padding(.top, .xxSmall)
             }
         }
+        .animation(.easeInOut(duration: 0.2))
```

Comment:
Animation without `value:` will be deprecated. Should be `, value: passwordStrength`

Reply:
One more thing I realized. If we bake-in the animation here, it will be out of sync with the parent stacks animations.
We probably should not animate the whole InputField at all, so that caller can decide if and how to animate the stacks.

Comparison:

Current version (out of sync animation)

https://user-images.githubusercontent.com/35844477/170255937-61c23a75-ec25-4697-9393-f63331ed9db2.mov

Proposed version (without this animation modifier and with applied .animation() to the specific preview VStack only)

https://user-images.githubusercontent.com/35844477/170256072-102d3c06-1c11-4bda-987f-7476f73a290b.mov